js-intro.js

```diff
@@ -47,10 +70,23 @@ var myMessage = "Learning is fun!"
 
 // 3a. Write the code that logs each letter of the message using a for loop.
 //hint: "logs" in this case means "prints to the terminal console."  This is typically done with a console.log(x), where x is the string, function, etc that you wish to see in the terminal.
-
+const learning = (string) => {
+    let newStr = myMessage
+    for(let i=0; i < string.length; i++){
+ }
+ return newStr
+}
+console.log(learning(myMessage))
 
```

Comment:
Right now you are just returning the full string. A few changes will get you each letter.
const learning = (string) => {
  for(let i=0; i < string.length; i++){
    console.log(string[i])
 }
}
learning(myMessage)